apps/web/nuxt.config.ts

```diff
@@ -71,6 +71,9 @@ export default defineNuxtConfig({
       showConfigurationDrawer: process.env.SHOW_CONFIGURATION_DRAWER === '1' ?? false,
       primaryColor: process.env.PRIMARY || '#0c7992',
       secondaryColor: process.env.SECONDARY || '#008ebd',
+      newsletterForm: process.env.NEWSLETTERFORM === undefined ? true : process.env.NEWSLETTERFORM === 'true',
+      newsletterFormNames:
+        process.env.NEWSLETTERFORMNAMES === undefined ? false : process.env.NEWSLETTERFORMNAMES === 'true',
```

Comment:
```suggestion
      newsletterForm: process.env.NEWSLETTERFORM === 'true' || true,
      newsletterFormNames: process.env.NEWSLETTERFORMNAMES === 'true' || false,
```

Reply:
I tested this approach and it is not working for all 3 cases. When the variables is set in the .env but it is set to 'false', it will just set it as true (because of || true)